apps/next/src/pages/AccountManagement/components/ImportPrivateKey/components/ImportPrivateKeyForm.tsx

```diff
@@ -0,0 +1,229 @@
+      );
+      if (findAccount) {
+        setIsKnownAccount(true);
+      }
+    },
+    [allAccounts, setIsKnownAccount],
+  );
+
+  const errorHandler = useCallback(
+    (errorMessage: string) => {
+      setDerivedAddresses(undefined);
+      setError(errorMessage);
+    },
+    [setDerivedAddresses, setError],
+  );
+
+  const validate = useCallback(
+    (key: string) => {
+      const validationError = t(
+        'The key you entered is invalid. Please try again',
+      );
+      const strippedPk = utils.strip0x(key);
+
+      if (strippedPk.length !== 64) {
+        errorHandler(validationError);
+        return;
+      }
+
+      try {
+        const publicKey = getPublicKeyFromPrivateKey(strippedPk);
+        const addressC = getEvmAddressFromPubKey(publicKey);
+        checkIfAccountExists(addressC);
+        const addressBTC = getBtcAddressFromPubKey(
+          publicKey,
+          network?.isTestnet ? networks.testnet : networks.bitcoin,
+        );
+        setDerivedAddresses({
+          addressC,
+          addressBTC,
+        });
+        setError('');
+      } catch (_err) {
+        errorHandler(validationError);
+      }
+    },
+    [
+      checkIfAccountExists,
+      errorHandler,
+      network?.isTestnet,
+      setDerivedAddresses,
+      setError,
+      t,
+    ],
+  );
+
+  const keyInputHandler = useCallback(
+    (event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+      const newKey = event.target.value;
+      setPrivateKey(newKey);
+
+      if (newKey) {
+        validate(newKey);
+      } else {
+        errorHandler(t('Please enter the private key.'));
+      }
+    },
+    [errorHandler, setPrivateKey, t, validate],
+  );
+
+  const readyToImport = derivedAddresses && !error && !isImportLoading;
+
+  const handleSubmit = useCallback(() => {
+    capture('ImportPrivateKeyConfirmClicked');
+    if (isKnownAccount) {
+      setIsDuplicatedAccountDialogOpen(true);
+      return;
+    }
+
+    return handleImport();
+  }, [isKnownAccount, setIsDuplicatedAccountDialogOpen, handleImport, capture]);
+
+  return (
+    <>
+      <Typography
+        variant="h2"
+        sx={{
+          mt: '23px',
+          mb: 6,
+          '&.MuiTypography-root': {
+            fontWeight: 700,
+          },
+        }}
+      >
+        {t('Import private key')}
+      </Typography>
+      <Stack>
+        <LessRoundedPasswordField
+          value={privateKey}
+          placeholder={t('Enter private key')}
+          onChange={keyInputHandler}
+          error={!!error}
+          helperText={error}
+        />
+
+        {!error && (
+          <Stack sx={{ mt: 2, rowGap: '10px' }}>
+            <DerivedAddressList
+              derivedAddresses={derivedAddresses}
+              isLoading={isImportLoading}
+            />
+
+            {derivedAddresses && (
+              <Card sx={{ p: 2 }}>
+                <Stack direction="row" sx={{ justifyContent: 'space-between' }}>
```

Comment:
Please try to be consistent in using MUI component API (sx vs system props)

```suggestion
                <Stack direction="row" justifyContent="space-between">
```
or

```suggestion
                <Stack sx={{ flexDirection: 'row', justifyContent: 'space-between' }}>
```